webdriver-spec.html

```diff
@@ -1211,9 +1211,9 @@
  </tr>
 
  <tr>
-  <td><dfn>invalid element coordinates</dfn>
+  <td><dfn>invalid coordinates</dfn>
   <td>400
-  <td><code>invalid element coordinates</code>
+  <td><code>invalid coordinates</code>
```

Comment:
So I feel like this section is already so backward incompatible that making the name of the error code correct (since there is no element involved) is totally reasonable. Note that this error is not currently used elsewhere in the specification.